docs/changelog.md

```diff
@@ -4,9 +4,19 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.5. - Dec 7, 2022
+
+### Bug Fixes
+
+#### Graph modules
+
+- The MAGE C++ API is now consistent with standard library classes. We fixed bugs related to constructors and assignment operators, and corrected the API type check in `ValueNumeric`.
```

Comment:
This is again - what you did, rather than what the user got from it... 

Maybe:
Constructors and assignment operators now work as expected, and the API type check in the WHAT `ValueNumeric`now DOES WHAT?

Reply:
Oh, I see now! Fixed.